tests/functional/lib/helpers.js

```diff
@@ -1004,6 +1004,26 @@ define([
     };
   }
 
+  function testAttribute (elementSelector, attributeName, assertion, value) {
```

Comment:
> Do you think we should use the `testElement*` convention since these tests relate to elements?

I'm happy to, although it seems tautological to me. An attribute is always on an element by definition. So the full names should be `testElementAttribute`, `testElementAttributeEquals` and `testElementAttributeMatches`, yep?